frontend/public/components/utils/copy-to-clipboard.jsx

```diff
@@ -1,15 +1,65 @@
 import * as React from 'react';
 
 import { CopyToClipboard as CTC } from 'react-copy-to-clipboard';
+import { Tooltip } from 'react-lightweight-tooltip';
 
-export const CopyToClipboard = ({value, visibleValue = value}) => <React.Fragment>
-  <div className="co-copy-to-clipboard">
-    <pre className="co-pre-wrap co-copy-to-clipboard__text">{visibleValue}</pre>
-    <CTC text={value}>
-      <button className="btn btn-default co-copy-to-clipboard__btn" type="button">
-        <i className="fa fa-clipboard" aria-hidden="true"></i>
-        <span className="sr-only">Copy to Clipboard</span>
-      </button>
-    </CTC>
-  </div>
-</React.Fragment>;
+export class CopyToClipboard extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {copied: false};
+  }
+
+  render() {
+    const overrides = {
```

Comment:
If this can be reused in other contexts for a left-facing tooltip, I'd move it to tooltip.jsx as another option (top by default, but left position either as a prop or another named component)